frontend/components/organisms/CourseDetailVideoSection.tsx

```diff
@@ -90,6 +91,7 @@ export function CourseDetailVideoSection({
               <Spacer />
               {session?.user?.id && selectedVideo && (
                 <>
+                  <GoodButton />
```

Comment:
```suggestion
                  <GoodButton userId={userId} videoId={videoId} />
```

的な形で
GoodButton
で求められる引数を渡してあげる必要がありそうですね！

Reply:
@suzury 
これって確認できてます？？

エラー文の通りなのですが
  isLiked,
  handleLike,
  likeCount,
を受け取るように実装しているのに呼び出し元で値を渡してあげていないからエラーが出てます。
なので全てここで渡してあげるように実装してください。